modules/cluster/outputs.tf

```diff
@@ -11,6 +11,14 @@ output "cluster_oidc_issuer_url" {
   value = local.oidc_provider_url
 }
 
+// ----------------------------------------------------------------------------
+// VPC Outputs
+// ----------------------------------------------------------------------------
+output "vpc" {
+  description = "All outputs from the VPC module"
+  value       = module.vpc
```

Comment:
https://github.com/terraform-aws-modules/terraform-aws-vpc/blob/master/outputs.tf -> This is the list of output that you can use from. So if you just want the id, `module.vpc.id`